lib/angular2/shared/effects/model.ejs

```diff
@@ -0,0 +1,116 @@
+/* tslint:disable */
+import 'rxjs/add/operator/map';
+import 'rxjs/add/operator/catch';
+import 'rxjs/add/operator/startWith';
+import 'rxjs/add/operator/switchMap';
+import 'rxjs/add/operator/mergeMap';
+import 'rxjs/add/operator/toArray';
+import { Observable } from 'rxjs/Observable';
+import { of } from 'rxjs/observable/of';
+import { concat } from 'rxjs/observable/concat';
+import { Injectable, Inject } from '@angular/core';
+import { Effect, Actions } from '@ngrx/effects';
+
+import { LoopbackAction } from '../models/BaseModels';
+import { BaseLoopbackEffects } from './base';
+
+import { <%- modelName %>ActionTypes, <%- modelName %>Actions } from '../actions/<%- modelName %>';
+import { LoopbackErrorActions } from '../actions/error';
+import { <%- modelName %>Api } from '../services/index';
+
+@Injectable()
+export class <%- modelName %>Effects extends BaseLoopbackEffects {
+<% model.methods.forEach(function(action) {
+ 
+  var methodName = action.name.split('.').join('$').replace('prototype$', '').replace(/::/g, '__');
```

Comment:
Same here a lot of repeated code, not only in actions but this effects model is also copied from the services one, now makes much more sense to create a helper to provide the repeated code.